syntax = "proto3";

package transport_catalogue_model;

message RoutingSettings {
    uint32 bus_wait_time = 1;
    double bus_velocity = 2;
}